bpf/headers/profile.h

```diff
@@ -0,0 +1,27 @@
+#ifndef __PROFILE_H_
```

Comment:
This should probably be named errors.h and be in the main directory. The helpers are what we get from ebpf-go and libbpf